app/components/elements/forms/text_field_component.rb

```diff
@@ -6,8 +6,11 @@ module Forms
     class TextFieldComponent < FieldComponent
       def initialize(**args)
         args[:container_classes] = merge_classes('field-container', args[:container_classes])
+        @error_classes = args.delete(:error_classes) || []
```

Comment:
This avoided having to add another argument to `FieldComponent`.

Reply:
I'd suggest that there is no reason to avoid this.